boot/freeldr/freeldr/ntldr/winldr.c

```diff
@@ -583,14 +595,17 @@ LoadWindowsCore(IN USHORT OperatingSystemVersion,
              * Parse the port name.
              * Format: /DEBUGPORT=COM[1-9]
              * or: /DEBUGPORT=FILE:\Device\HarddiskX\PartitionY\debug.log
+             * or: /DEBUGPORT=SCREEN
              * or: /DEBUGPORT=FOO
```

Comment:
These are **just** examples, not the only actual list of options supported. (and "FOO" is definitively placeholder for any name).